features/apt_messages.feature

```diff
@@ -113,14 +113,14 @@ Feature: APT Messages
         Building dependency tree...
         Reading state information...
         Calculating upgrade...
+        The following security updates require Ubuntu Pro with 'esm-infra' enabled:
+          .*
+        Learn more about Ubuntu Pro for 16\.04 at https:\/\/ubuntu\.com\/16-04
```

Comment:
I moved this to the top so that it would be next to the similar messaging around packages that need an `apt autoremove`

Reply:
Should we have another call in this(these maybe) test(s) to ensure that the message is not shown in `apt-get`?

Reply:
Good idea! I'll add one in an extra commit

Reply:
updated!